intellij-ext/src/main/kotlin/com/metalbear/mirrord/LogLevel.kt

```diff
@@ -0,0 +1,5 @@
+package com.metalbear.mirrord
+
+enum class LogLevel {
+    ERROR, WARN, INFO, DEBUG, TRACE;
+}
```

Comment:
The reason we kept this as a text field was to allow the user to choose whatever logs they wanna see, forexample, 
RUST_LOG=mirrord=debug, warn

Reply:
@eyalb181 wdyt? should we keep this as a dropdown or a text field

Reply:
We're probably going to change how our logs are configured anyway (see #390 ), but for now lets keep it as a dropdown, it looks much better and doesn't limit functionality by much.